src/GitHub.InlineReviews/Services/PullRequestSessionService.cs

```diff
@@ -517,7 +517,7 @@ public ITextDocument GetDocument(ITextBuffer buffer)
                     OriginalCommitId = x.Comment.OriginalCommit.Oid,
                     PullRequestReviewId = x.Comment.PullRequestReview.DatabaseId.Value,
                     User = user,
-                    IsPending = true,
+                    IsPending = false,
```

Comment:
Are you sure this is right? This method is `PostPendingReviewCommentReply`, so doesn't this need to be set to pending?

Reply:
Thanks